app/views/govuk_publishing_components/components/docs/intervention.yml

```diff
@@ -38,6 +38,17 @@ examples:
       suggestion_link_text: "You can now apply for a permit online."
       suggestion_link_url: "/permit"
 
+  open_suggestion_link_in_new_tab:
+    description: |
+      For example if you wanted to drive users to another online task, you don't want to completely hijack
+      their original flow on GOV.UK. Adding the option to open links in a new tab helps to address this.
+      Link text should tell the user that the link opens in a new tab.
+      Note: "(opens in a new tab)" is added to link text if the phrase isn't included.
+    data:
+      suggestion_link_text: "You can now apply for a permit online (opens in a new tab)"
```

Comment:
Maybe change this line to just `"You can now apply for a permit online"` to demonstrate the auto text appending?